meson.build

```diff
@@ -33,6 +33,19 @@ example_deps = [gst_dep, gst_base_dep]
 # Define test dependencies
 test_deps=[gst_dep, gst_base_dep, gst_check_dep]
 
+# Verify if profiling was enabled
+if get_option('profiling').enabled()
+  profiler_dep = dependency('libprofiler')
+  if(profiler_dep.found())
+    message('Profiling enabled: Building examples with profiling support.')
+    link_flags = ['-lprofiler']
+    # Update test test_deps to include profiler dependency
+    test_deps=[gst_dep, gst_base_dep, gst_check_dep, profiler_dep]
```

Comment:
Can we add only the new dependency to the list of default dependencies?
For example
`test_deps+=profiler_dep`